shimmingtoolbox/masking/mask_mrs.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Creating MRS mask API
+"""
+import numpy as np
+import numpy.linalg as npl
+import nibabel as nib
+import logging
+from shimmingtoolbox.utils import splitext
+from shimmingtoolbox.utils import run_subprocess
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def mask_mrs(fname_input, raw_data, center, size):
+    """
+    Create a mask to shim single voxel MRS
+
+    Args:
+        fname_input (str): Input path of the fieldmap to be shimmed (supported extention .nii and .nii.gz)
+        raw_data (str): Input path of the of the twix raw-data (supported extention .dat)
+        center (list): voxel's center position in mm of the x, y and z of the scanner's coordinate
+        size (list): voxel size in mm of the x, y and z of the scanner's coordinate
+    Returns:
+        numpy.ndarray: Cubic mask with same dimensions as MRS voxel.
+    """
+
+    if fname_input is None:
+        raise TypeError(f"The fname_input cannot be None.")
+
+    if raw_data is None:
+        logger.info(" The raw_data is not provided, creating the mask with the given voxel position and size info")
+        if center is None or size is None:
+            raise TypeError('The raw_data is not provided; the voxel position and size are required to proceed')
+        else:
+            mrs_voxel_size = size
+            logger.debug('center:', center)
+            scanner_coordinate = np.array(center + (1,))
```

Comment:
Add space between comma and following character
```suggestion
            scanner_coordinate = np.array(center + (1, ))
```